src/components/userGroups/User.tsx

```diff
@@ -93,6 +94,7 @@ export default function UserForm({
     function validateForm(): boolean {
         if (emailState.emails.length === 0) {
             setEmailState((emailState) => ({ ...emailState, emailError: 'Emails are mandatory.' }));
+            toast.error('Some required fields are missing')
```

Comment:
We are already showing an error message. Does it add any value?(Do confirm with utkarsh as wee)

Reply:
When the user hit save, the email input area is not visible to users as the form div height is greater than the parent div, hence showing toast to prevent confusion